### PR TITLE
[FE-278] Added USB flasher

### PR DESCRIPTION
**Pull request by gkoumanos**
Page should only be visible with direct link and not in sidebar.
Download link will be publicly available soon.

### CODE REVIEW

File: docs/wxm-devices/wifi-m5-bundle/m5-usb-flasher.mdx [hunk 1]
```diff
@@ -0,0 +1,113 @@
+---
+id: m5-usb-flasher
+title: Updating Your M5 Miner Firmware with Flasher Tool
+slug: /wxm-devices/wifi-m5-bundle/m5-usb-flasher
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+<h2 id="introduction">Introduction</h2>
+<p>If you are currently using an M5 Miner with firmware version 0.3.11, it's essential to update your device using the Flasher Tool. This version is considered obsolete and cannot receive updates through the "Over The Air" (OTA) method due to significant differences in the underlying operating system. To ensure your M5 Miner continues to operate efficiently and receive rewards, you must perform a full firmware flash using a USB cable.</p>
```
Comment (PavlosTze) [inline docs/wxm-devices/wifi-m5-bundle/m5-usb-flasher.mdx:10]: an M5 Miner -> a M5 miner

File: docs/wxm-devices/wifi-m5-bundle/m5-usb-flasher.mdx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+<p>This guide will walk you through the step-by-step process of updating your M5 Miner to the latest firmware version, ensuring you stay up to date with the latest features, security enhancements and continue to receive rewards.</p>
+<h2 id="how-to-update-your-m5-miner">How to Update Your M5 Miner</h2>
+<p>Follow these instructions carefully to update your M5 Miner's firmware using the Flasher Tool. We've included screenshots to help you navigate the process.</p>
+<p><strong>Step 1: Prepare Your Equipment</strong></p>
+<p>Before starting, ensure you have the following:</p>
+<ul>
+<li>An M5 Miner powered on with firmware version 0.3.11</li>
```
Comment (PavlosTze) [inline docs/wxm-devices/wifi-m5-bundle/m5-usb-flasher.mdx:17]: An M5 Miner -> A M5 Miner

**Review by PavlosTze — CHANGES_REQUESTED** [review]
@gkoumanos  Except these 2 minor typos, everything looks good.

File: docs/wxm-devices/wifi-m5-bundle/m5-usb-flasher.mdx (hunk 1 above)
Comment (gkoumanos) [reply docs/wxm-devices/wifi-m5-bundle/m5-usb-flasher.mdx:10]: begins with the consonant letter M, but the M is pronounced with its letter name, "em."
https://www.britannica.com/dictionary/eb/qa/how-do-you-know-whether-to-use-a-or-an

File: docs/wxm-devices/wifi-m5-bundle/m5-usb-flasher.mdx (hunk 2 above)
Comment (gkoumanos) [reply docs/wxm-devices/wifi-m5-bundle/m5-usb-flasher.mdx:17]: begins with the consonant letter M, but the M is pronounced with its letter name, "em."
https://www.britannica.com/dictionary/eb/qa/how-do-you-know-whether-to-use-a-or-an

**Pull request merged**